src/patternfly/components/FormControl/examples/form-control-select-example.hbs

```diff
@@ -16,8 +16,8 @@
     <option value="Option 3">The third option</option>
     <option value="Option 4">The fourth option</option>
   </optgroup>
-{{/form-controls}}
-{{#> form-controls controlType="select" pf-c-form-control--modifier="pf-m-invalid" required="true" aria-errormessage="select2ErrorText" id="select2" name="selectexample2" aria-label="select group example"}}
+{{/form-control}}
+{{#> form-control controlType="select" required="true" aria-invalid="true" id="selectExample3" name="selectExample3" aria-label="select group example"}}
```

Comment:
the `aria-label` should indicate this is an error state to be consistent with the other examples.